gulpfile.js

```diff
@@ -31,7 +31,7 @@ gulp.task('bootstrap', async () => {
   })
 })
 
-for (let name of ['preprocessors', 'parsers', 'prefixers', 'tokenizers', 'linters']) {
+for (let name of ['preprocessors', 'parsers', 'prefixers', 'tokenizers', 'linters', 'sourcemaps']) {
```

Comment:
Could you run Prettier (in VSCode or by installing it globally).

We have an 80 columns limit.

Reply:
I tried with the VSCode extension and by installing globally but I couldn't get it to wrap these lines.

Is it possible that you have a global/default config for prettier?

Reply:
I have config in `package.json`.

Don’t worry, I will fix it after merging.

Reply:
thank you 🙇